packages/matchbox/src/components/Inline/Inline.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Box } from '../Box';
+import styled from 'styled-components';
+import propTypes from '@styled-system/prop-types';
+import { negativeTop, negativeLeft, alignChildren } from './styles';
+
+// Negates children padding top
+const OuterWrapper = styled('div')`
+  padding-top: 1px;
```

Comment:
Maybe you know of a better way to do this, but basically doing a negative margin top here will also move the parent container with it. Adding an extra box with height forces negative margin to only apply to children

Reply:
Just so our side convo is made more public, here is a Codepen using `overflow: auto;` instead of `padding-top` to handle this problem. *Slightly* less hacky but still weird?

https://codepen.io/nicklemmon/pen/ZEYVedY?editors=1100

One or the other appears to be required, though.